docs/Tooling/stacked-cli.md

```diff
@@ -124,18 +124,18 @@ Underneath your last registration in `registerServices` add
 // @stacked-mock-register
 ```
 
-Now you can run `stacked create service myService` and you'll see the files created plus all the registration happens.
```

Comment:
Same here as with myView, this is going to result in `MyServiceService` and could lead into confusion.

Reply:
I agree.